packages/gatsby/src/bootstrap/load-plugins.js

```diff
@@ -133,6 +133,9 @@ module.exports = async (config = {}) => {
   }
 
   // Add internal plugins
+  plugins.push(
+    processPlugin(path.join(__dirname, `../internal-plugins/dev-404-page`))
```

Comment:
This was related to the fix?

Reply:
I'm not quite sure, but examining the code lead me to the conclusion that the plugin loading order might be relevant.

Reply:
Perhaps, it would make the copying run earlier which might matter for some reason haha 👍